lib/ranked-model/ranker.rb

```diff
@@ -78,7 +78,9 @@ def position
       end
 
       def relative_rank
-        finder.where("#{ranker.column} < #{rank}").count(:all)
+        escaped_column = ActiveRecord::Base.connection.quote_column_name ranker.column
```

Comment:
Rather than reaching straight for the `ActiveRecord::Base` class is it possible to just access `connection.quote_column_name` perhaps off the `ranker`?

Reply:
Thanks for the review! I modified it to use `Mapper#instance_class.connection`, is my understanding correct?

Reply:
Yep that's perfect :) Ping me once the tests go green and I'll merge this :D